tests/Unit/Wire/IO/StreamIOTest.php

```diff
@@ -49,4 +49,17 @@ public function select_must_throw_io_exception()
 
         $stream->select(0, 0);
     }
+
+    /**
+     * @test
+     */
+    public function connect_ipv6()
+    {
+        $streamIO = new StreamIO(HOST6, PORT, 0.1, 0.1, null, false, 0);
+        $streamIO->connect();
+        $ready = $streamIO->select(0, 0);
+        $this->assertEquals(0, $ready);
+
+        return $streamIO;
```

Comment:
There is no need to return anything from test method. Just make sure all assertions are implemented.